pyspeckit/spectrum/plotters.py

```diff
@@ -39,6 +40,11 @@ def __init__(self, Spectrum, autorefresh=True, title="", ylabel="",
         self.figure = None
         self.axis = None
         self.Spectrum = Spectrum
+        self._xunit = Spectrum.xarr.unit
+        if self._xunit == 'microns':
+            self._xunit = 'micron'
+        elif self._xunit == 'angstroms':
+            self._xunit = 'Angstrom'
```

Comment:
Where are the plural versions of these units coming from?  They should just be replaced with their singular variants wherever needed.  If they really are necessary, we should use a universal `plural_units_mapping` dictionary {'angstroms':'angstrom'}  somewhere (in `units.py`?) instead of using two `if` statements.